src/app/config.rs

```diff
@@ -810,38 +810,26 @@ fn parse_dns_suffix(s: &str) -> Result<dns::Suffix, ParseError> {
 pub fn parse_backoff<S: Strings>(
     strings: &S,
     base: &str,
-    default: Backoff,
-) -> Result<Backoff, Error> {
-    let env_disabled = format!("LINKERD2_PROXY_{}_EXP_BACKOFF_DISABLED", base);
```

Comment:
Do we need to worry about still checking for this environment variable?

Reply:
Executive decision: no. If you really don't want a backoff you can set min=0/max=0